index.html

```diff
@@ -157,113 +178,9 @@ <h3 class="title-main">Main Program</h3>
-                <p class="title-bar-speaker lowbar-title"></p>
-                <article class="speaker-bio">
-                  Mayito has been one of the most prominent learders for the
-                  BUMSA community and the alumni community. He has done many
-                  tramendous things to ensure that BUMSA is a success.
-                </article>
-              </div>
-            </section>
-          </div>
-          <div>
-            <section class="speakers">
-              <img
-                src="./assets/images/speaker_01.png"
-                alt="speaker1"
-                class="img-speaker"
-              />
-              <div class="speaker-info">
-                <p class="name">Mayito Ismail</p>
-                <p class="speaker-title">Student leader</p>
-                <p class="title-bar-speaker lowbar-title"></p>
-                <article class="speaker-bio">
-                  Mayito has been one of the most prominent learders for the
-                  BUMSA community and the alumni community. He has done many
-                  tramendous things to ensure that BUMSA is a success.
-                </article>
-              </div>
-            </section>
-            <section class="speakers">
-              <img
-                src="./assets/images/speaker_01.png"
-                alt="speaker1"
-                class="img-speaker"
-              />
-              <div class="speaker-info">
-                <p class="name">Mayito Ismail</p>
-                <p class="speaker-title">Student leader</p>
-                <p class="title-bar-speaker lowbar-title"></p>
-                <article class="speaker-bio">
-                  Mayito has been one of the most prominent learders for the
-                  BUMSA community and the alumni community. He has done many
-                  tramendous things to ensure that BUMSA is a success.
-                </article>
-              </div>
-            </section>
-            <section class="speakers">
-              <img
-                src="./assets/images/speaker_01.png"
-                alt="speaker1"
-                class="img-speaker"
-              />
-              <div class="speaker-info">
-                <p class="name">Mayito Ismail</p>
-                <p class="speaker-title">Student leader</p>
-                <p class="title-bar-speaker lowbar-title"></p>
-                <article class="speaker-bio">
-                  Mayito has been one of the most prominent learders for the
-                  BUMSA community and the alumni community. He has done many
-                  tramendous things to ensure that BUMSA is a success.
-                </article>
-              </div>
-            </section>
-          </div>
+        <section class="speaker-container" id="speaker-container">
+          <div></div>
+          <div></div>
         </section>
       </section>
       <footer class="footer">
```

Comment:
- [ ] Kindly make sure to add some space between the logo and text in the footer area, so the website looks identical to the provided design

|Youe website|Provided design|
|-|-|
|![Screenshot_20230322_090155](https://user-images.githubusercontent.com/25227022/226851866-59e18730-fd0d-4024-8c5c-3c31c8360bc8.png)|![Screenshot_20230322_090345](https://user-images.githubusercontent.com/25227022/226852263-d987a4e5-344d-454c-8b43-6a6f1196eaaf.png)|